infrastructure/cdn-in-a-box/optional/docker-compose.debugging.yml

```diff
@@ -81,16 +72,6 @@ services:
       args:
```

Comment:
Same as above: The `-go` in `trafficops-go-nondebug`, `trafficops-go`, `trafficops-go-debug`, and `Dockerfile-go` can be removed.